website/docs/cloud-docs/vcs/azure-devops-server.mdx

```diff
@@ -25,9 +25,9 @@ Terraform Cloud uses personal access tokens to connect to Azure DevOps Server. T
 
    If you just created your organization, you might already be on this page. Otherwise:
 
-   1. Make sure the upper-left organization menu currently shows your organization.
-   1. Click the "Settings" link at the top of the page (or within the ☰ menu)
-   1. On the next page, click "VCS Providers" in the left sidebar
+   1. Make sure the bottom left organization menu currently shows your organization.
+   1. Click "Settings".
+   1. On the next page, click "VCS Providers"
    1. Click the "Add a VCS Provider" button
```

Comment:
Please replace all of this, including everything in Step 1 with the following:

   1. Go to your organization's settings and then click **Providers**. The **VCS Providers** page appears.
   1. Click **Add a VCS Provider**. The **Add VCS Provider** page appears.